Update of section loading a delimited text file (#4652)
qgis/QGIS-Documentation: source/docs/user_manual/managing_data_source/opening_data.rst

Importing a delimited text file
-------------------------------

> Delimited text file (e.g. :file:`.txt`,:file:`.csv`, :file:`.dat`, :file: `.wkt`) 

Reviewer: ```suggestion
Delimited text files (e.g. :file:`.txt`, :file:`.csv`, :file:`.dat`, :file: `.wkt`) 
```

can be loaded in QGIS using the tools described above. However, loaded this way,
it'll show up like a simple table data. Sometimes, delimited text files can contain

> geometric data you'd want to visualize; this is what the |addDelimitedTextLayer|:guilabel:`Add

Reviewer: ```suggestion
coordinates / geometries that you could want to visualize; this is what the
|addDelimitedTextLayer|:guilabel:`Add Delimited Text Layer` is designed for.
```

> Delimited Text Layer` is designed for.

Reviewer: ```suggestion
```

Once the file is parsed, set :guilabel:`Geometry definition` to

* |radioButtonOn|:guilabel:`Point coordinates` and provide the :guilabel:`X
  field`, the :guilabel:`Y field`, :guilabel:`Z field` (for 3-dimensional data)

>   and the :guilabel:`M field`(for the measurement dimension) if the layer is of 

Reviewer 2: ```suggestion
  and the :guilabel:`M field` (for the measurement dimension) if the layer is of 
```

  point geometry type and contains such coordinate fields. If the coordinates
  are defined as degrees/minutes/seconds, activate the

>   |checkbox|:guilabel:`DMS coordinates`checkbox;

Reviewer: ```suggestion
  |checkbox|:guilabel:`DMS coordinates` checkbox;
```

Reviewer: Could this be the cause of the travis problem?

Reviewer 2: Yes, that's (actually the one I highlighted, but this one would be next one if not fixed)
